Makefile

```diff
@@ -0,0 +1,19 @@
+CRYSTAL_BIN ?= $(shell which crystal)
+PREFIX ?= /usr/local
+SHARD_BIN ?= ../../bin
+
+build:
+	$(CRYSTAL_BIN) build --no-debug -o bin/neph src/bin/neph_bin.cr $(CRFLAGS)
+clean:
+	rm -f ./bin/neph
+install: build
+	mkdir -p $(PREFIX)/bin
+	cp ./bin/neph $(PREFIX)/bin
+bin: build
+	mkdir -p $(SHARD_BIN)
+	cp ./bin/neph $(SHARD_BIN)
+test: build
+	$(CRYSTAL_BIN) spec
+	./bin/neph
```

Comment:
This would build the binary again, and also generate the `man` page. (And `crystal spec` also builds the binary.) Is a `man` page needed for testing the code?
Also, if this `Makefile` is intended to build a `shard` dependency, I think this job is unneeded. If the repo is cloned directly, Neph can be used for the `test` job.